cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -367,28 +361,21 @@ func (s *mysqlBackend) execDMLWithMaxRetries(ctx context.Context, dmls *prepared
 		retry.WithIsRetryableErr(isRetryableDMLError))
 }
 
-// Only for testing.
-func (s *mysqlBackend) setDMLMaxRetry(maxRetry uint64) {
-	s.dmlMaxRetry = maxRetry
-}
-
 func logDMLTxnErr(
-	err error, start time.Time, changefeed model.ChangeFeedID,
+	err error, start time.Time, changefeed string,
 	query string, count int, startTs []model.Ts,
 ) error {
 	if isRetryableDMLError(err) {
 		log.Warn("execute DMLs with error, retry later",
 			zap.Error(err), zap.Duration("duration", time.Since(start)),
 			zap.String("query", query), zap.Int("count", count),
 			zap.Uint64s("startTs", startTs),
-			zap.String("namespace", changefeed.Namespace),
-			zap.String("changefeed", changefeed.ID))
+			zap.String("changefeed", changefeed))
```

Comment:
This is different from our other logs.

Reply:
I think both are OK so I prefer to let it be.

Reply:
It might be better to unify the log structure. No matter, we need to add string method for changefeedID to unify it. We can modify it in the future.